src/formatDate.js

```diff
@@ -50,7 +50,37 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.slice(-1);
+  const newSeparator = toFormat.slice(-1);
+  const copyDate = date.split(separator);
+  const copyFromFormat = fromFormat.slice(0, 3);
+  const copyToFormat = toFormat.slice(0, 3);
```

Comment:
Why do you need these array copies?
You can use `fromFormat` and `toFormat` directly by using cycle like this:
```
for (let i = 0; i < toFormat.length - 1; i++)
```